apps/dashboard/src/lib/utils/functions/date.spec.ts

```diff
@@ -0,0 +1,37 @@
+import '@testing-library/jest-dom';
+import { calDateDiff, getGreeting } from './date';
+
+describe('calDateDiff', () => {
+  test('should return the correct time difference when the input date is in the past', () => {
+    const inputDate = new Date('2022-01-01');
+    const expectedOutput = '2 years ago';
+
+    const result = calDateDiff(inputDate);
+
+    expect(result).toBe(expectedOutput);
+  });
+
+  test('should return the correct time difference when the input date is a number', () => {
+    const inputDate = 1640995200000; // January 1, 2022
+    const expectedOutput = '2 years ago';
+
+    const result = calDateDiff(inputDate);
+
```

Comment:
@Toyin5 I think you might need to set the system time (like you are doing in the getGreeting test suite) because after a year this test will start failing.

Reply:
Ok. Will do that